lib/errinj.c

```diff
@@ -0,0 +1,104 @@
+/**
+ *		Tempesta FW
+ *
+ * Error injection library.
+ *
+ * Application protocol handler layers must implement zero data copy logic
+ * on top on native Linux socket buffers. The helpers provide common and
+ * convenient wrappers for skb processing.
```

Comment:
I didn't get the comment - why error injection file comments something about skb? A dirty copy & paste? BTW it'd be good to write some description about the error injection design, maybe some usage example

Reply:
Fixed